### PR TITLE
text-summaNum-consolClick

### PR DESCRIPTION
**Pull request by AndreyIvchenko**
Привет. 
Я себя чувствую как старый ржавый МАЗ, который потиху пытается трогаться с места. )

### CODE REVIEW

File: scripts/landingPage.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function showString() {
```
Comment (kirillstarovoitov) [inline scripts/landingPage.js:1]: this func should receive a param. At this implementation no params. Fix it pls

File: index.html [hunk 2]
```diff
@@ -11,10 +11,11 @@
 
 <body>
     <div class="divButton">
-        <button class="buttonSwitch">Switch color</button>
+        <button class="buttonSwitch" id="but1">Switch color</button>
     </div>
     <div style="width: 100%; height: 200px;"></div>
```
Comment (kirillstarovoitov) [inline index.html:16]: Whi we are using inline styles and for what this div is here?

File: scripts/landingPage.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    let a = "text"
+    console.log(a)
+}
+showString()
+
+
+function summaNum(n1, n2) {
```
Comment (kirillstarovoitov) [inline scripts/landingPage.js:8]: pls write the result in a variable then console.log it

File: scripts/landingPage.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    return n1 + n2;
+}
+console.log(summaNum(5, 5))
+
+document.getElementById('but1').addEventListener('click', butClick)
```
Comment (kirillstarovoitov) [inline scripts/landingPage.js:13]: pls write document.getElementById('but1') in variable (then u can console this variable and investigate result). Then add eventListener

File: scripts/landingPage.js (hunk 1 above)
Comment (kirillstarovoitov) [inline scripts/landingPage.js:1]: why we are using function declaration in this case?

**Review by kirillstarovoitov — CHANGES_REQUESTED** [review]

File: scripts/landingPage.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+function showCons(blabla) {
+    console.log('' + blabla)
+}
+showCons(1)
```
Comment (kirillstarovoitov) [inline scripts/landingPage.js:4]: Task is: 4. Написать 2 функции, первая принимает параметр в виде строки и выводит консоль лог эту строку при вызове.
The first function should receive a string (not another data type). In this case, you pass the number and then (inside a function) transform it to a string. It is the wrong solution. Let's improve this task. If param type is a string - then console log this string. In another case - console log text "Wrong data type". And pls renate param from blabla to something human understandable (eg string or param or argument)

File: scripts/landingPage.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+function summaNum(n1, n2) {
+    return n1 + n2;
```
Comment (kirillstarovoitov) [inline scripts/landingPage.js:7]: Improve this function. return sum if numbers only then both arguments are numbers. For another case - return null and console log text "one or both params, not numbers"

File: scripts/landingPage.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+}
+const res = summaNum(2.5, 2.5);
+console.log(res)
+
+let bid1 = document.getElementById("but1")
```
Comment (kirillstarovoitov) [inline scripts/landingPage.js:12]: It is work. But call variable smth like button or .. u know.. All developers should understand what contains this variable.

File: index.html [hunk 8]
```diff
@@ -11,10 +11,11 @@
 
 <body>
     <div class="divButton">
-        <button class="buttonSwitch">Switch color</button>
+        <button class="buttonSwitch" id="but1">Switch color</button>
     </div>
-    <div style="width: 100%; height: 200px;"></div>
+    <div></div>
```
Comment (kirillstarovoitov) [inline index.html:16]: this div is empty... Just remove it pls)

**Review by kirillstarovoitov — CHANGES_REQUESTED** [review]

**Comment by AndreyIvchenko** [comment]
done

**Pull request merged**